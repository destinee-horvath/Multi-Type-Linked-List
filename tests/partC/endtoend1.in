NEW 3
it
raining
tacos
NEW 4
cos
i
super
shy
new 100000
NEW 2
money
money
NEW 3
nested
{0}
{1}
VIEW 0
i wanna view a list
VIEW 3
TYPE 3
VIEW-NESTED 3
VIEW ALL
REMOVE 2
TYPE 1
INSERT 1 -1 eeee
NEW 4
me
n
my
gorls
DELETE 4 1
INSERT 4 1 and
VIEW-NESTED 4
VIEW NESTED list