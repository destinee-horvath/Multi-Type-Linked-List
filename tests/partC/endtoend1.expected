List 0: it -> raining -> tacos
List 1: cos -> i -> super -> shy
INVALID COMMAND: INPUT
List 2: money -> money
Nested 3: nested -> {List 0} -> {List 1}
it -> raining -> tacos
INVALID COMMAND: INPUT
nested -> {List 0} -> {List 1}
string -> reference -> reference
nested -> {it -> raining -> tacos} -> {cos -> i -> super -> shy}
Number of lists: 4
List 0
List 1
List 2
Nested 3
List 2 has been removed.

Number of lists: 3
List 0
List 1
Nested 3
string -> char -> string -> string
List 1: cos -> i -> super -> shy -> eeee
List 4: me -> n -> my -> gorls
List 4: me -> my -> gorls
List 4: me -> and -> my -> gorls
me -> and -> my -> gorls
INVALID COMMAND: VIEW
